spec/lib/rmt/cli/import_spec.rb

```diff
@@ -18,13 +19,19 @@
     end
 
     context 'with existing lockfile' do
-      before { allow(RMT::Lockfile).to receive(:create_file).and_raise(RMT::Lockfile::ExecutionLockedError) }
+      before do
+        allow(RMT::Lockfile).to receive(:create_file).and_raise(RMT::Lockfile::ExecutionLockedError)
```

Comment:
Is it possible to really create a lockfile? So that it actually executes the real check if a file is locked or not?
Just raising this execption skippes most of the code that we should test.

Reply:
so I changed it stubbing the the `flock` method. if you mean using real filesystem though I would disagree to do so. also in other specs we use `FakeFS`